### PR TITLE
Add a precision for references issues in doc

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @steveklabnik

### CODE REVIEW

File: src/doc/trpl/references-and-borrowing.md [hunk 1]
```diff
@@ -334,3 +334,35 @@ In other words, `y` is only valid for the scope where `x` exists. As soon as
...
+let y: &i32;
+let x = 5;
+y = &x;
+
+println!("{}", y);
+```
+
+We get this error:
+
+error: `x` does not live long enough
```
Comment (steveklabnik) [inline src/doc/trpl/references-and-borrowing.md]: needs triple ticks and `text`

**Comment by steveklabnik** [comment]
r= me after the nit

**Comment by GuillaumeGomez** [comment]
I saw there were also missing backticks at line 316, I added them. In this, should I squash ?

**Comment by GuillaumeGomez** [comment]
r= @steveklabnik

**Comment by steveklabnik** [comment]
The `r=` only works if you have commit bit :)

Anyway, I think that I have already accepted a PR to add that other missing one, but let's just let homu sort it out :)

**Comment by GuillaumeGomez** [comment]
Ok haha. I wasn't sure about the r= you, so I tried :p

**Pull request merged**